yelp_scrape.py

```diff
@@ -63,12 +64,40 @@ def collect_webpages(soup: BeautifulSoup, search_item: str) -> set:
     biz_lst: list = []
     for a in soup.find_all('a'):
         if 'href' in a.attrs:
-            if a.attrs['href'].split('?')[-1] == 'osq=' + search_item:
+            if a.attrs['href'].split('?')[-1] == 'osq=' + search_item and a.attrs['href'] not in biz_lst:
                 biz_lst.append(a.attrs['href'])
-    return set(biz_lst)
-
-
+        if len(biz_lst) >= 5:
+            break
+    return biz_lst
 
+def page_info_grab(pages: list) -> dict:
+    '''
+    iterate through the top search results obtained from "collect_webpages()" and pull relevant data to place into reccommndation info
+    :param list pages: list of urls of the top 5 search results from yelp
```

Comment:
this documents more where the function is _called from_ rather than what it _does_

nothing about this function requires that there are exactly 5 elements, or that they are the top elements
it just takes a list of Yelp URLs and a dictionary of stuff